src/cli/telemetry/test/telemetry.spec.ts

```diff
@@ -148,25 +148,21 @@ describe('prepareData', () => {
   let sys: d.CompilerSystem;
 
   beforeEach(() => {
-    config = {
-      outputTargets: [],
-      flags: createConfigFlags(),
-      logger: mockLogger(),
-    };
-
-    sys = createSystem();
+    config = mockValidatedConfig();
+    sys = config.sys;
+    // set static name + versions, otherwise tests will pull in the dev build's data (which changes per build)
+    sys.name = 'in-memory';
+    sys.version = '__VERSION:STENCIL__';
   });
 
-  it('provides an object', async () => {
+  it('prepares an object to send to ionic', async () => {
     const data = await telemetry.prepareData(coreCompiler, config, sys, 1000);
     expect(data).toEqual({
       arguments: [],
       build: coreCompiler.buildId,
       component_count: undefined,
-      config: {
-        flags: createConfigFlags(),
-        outputTargets: [],
-      },
+      // the configuration generation is tested elsewhere, just verify we're sending something under this flag
+      config: expect.any(Object),
```

Comment:
We discussed this a bit last Friday, but as the `ValidatedConfig` entity changes, I don't want to have to update this field every time. Because we test the anonymization of the config further down in this file, that _felt_ like adequate coverage to me